lib/dry/web/roda/skeletons/umbrella/Gemfile

```diff
@@ -0,0 +1,40 @@
+source "https://rubygems.org"
+
+gem "rake"
+
+# Web framework
+gem "dry-component"
+gem "dry-web"
+gem "dry-web-roda"
```

Comment:
should be from github (master) not gem. In current form you cannot generate subapp because stable version do not know how